assessment/collectors/app_migration_analysis.go

```diff
@@ -92,10 +104,24 @@ func NewMigrationSummarizer(ctx context.Context, googleGenerativeAIAPIKey *strin
 		return nil, fmt.Errorf("failed to create Vertex AI client: %w", err)
 	}
 
-	conceptExampleDB, err := assessment.NewMysqlConceptDb(projectID, location)
+	conceptExampleDB, err := assessment.NewMysqlConceptDb(projectID, location, language)
 	if err != nil {
 		return nil, fmt.Errorf("failed to load code example DB: %w", err)
 	}
+
+	var sourceFramework, targetFramework string
+
+	switch language {
+	case "go":
```

Comment:
The frameworks should be detected. If that is not possible now please add a TODO here.

Reply:
yes we can detect but the cleaner approach would be take it as input from user as part of `AssessmentConfig`